pwa-devdocs/src/peregrine/testing/storybook/index.md

```diff
@@ -2,10 +2,10 @@
 title: Storybook testing
 ---
 
-[Storybook][] is an independent application that lets you view and test UI components outside of a project.
-It also serves as a library for all the UI components in a project and how they look in isolation under different conditions.
+[Storybook][] is an independent application that lets you view and test React components outside of a project.
+It also serves as a library for all the visual components in a project and how they look in isolation under different conditions.
 
-Currently, the Peregrine package is the only project set up to use Storybook as a development and testing environment for its UI components.
+Currently, the Peregrine package is the only project set up to use Storybook as a development and testing environment for its components.
```

Comment:
This is not actually true, [Venia has a storybook too](https://github.com/magento-research/pwa-studio/blob/develop/package.json#L33).

Reply:
Ah, I see. This topic needs a revision anyways. Will add it to the backlog.